dbt_copilot_helper/templates/pipeline/buildspec.yml

```diff
@@ -0,0 +1,65 @@
+# {% version_info %}
+# Buildspec runs in the build stage of your environment pipeline to generate the environment CloudFormation stack config.
+version: 0.2
+env:
+  variables:
+    COPILOT_VERSION: 1.31.0
```

Comment:
Interesting idea, we could get the latest 1.x version of AWS Copilot and dyff dynamically so we are not using too stale a version of each.

Might be a risk that we introduce instability without it being obvious what changed to make things go wrong.

It would be interesting to see what the others think 👂

Reply:
I think that it would be fine to pass the version in when the tool is run. If we are running this for the first time for a project, we'd probably want the latest version anyway. If we're updating existing pipelines, then the version change would show in the diff.

Reply:
Raised a ticket for this: https://uktrade.atlassian.net/browse/DBTP-513

Reply:
Added a comment on the above ticket about also including this in the check config command and helper.